app/views/tag_aliases/_listing.html.erb

```diff
@@ -1,44 +1,34 @@
-<table width="100%" class="striped">
-  <thead>
-    <tr>
-      <th width="25%">From</th>
-      <th width="25%">To</th>
-      <th width="10%">Reference</th>
-      <th width="15%">Approver</th>
-      <th width="5%">Status</th>
-      <th width="15%"></th>
-    </tr>
-  </thead>
-  <tbody>
-    <% tag_aliases.each do |tag_alias| %>
-      <tr id="tag-alias-<%= tag_alias.id %>">
-        <td class="category-<%= tag_alias.antecedent_tag.try(:category) %>"><%= link_to tag_alias.antecedent_name, posts_path(:tags => tag_alias.antecedent_name) %> <span class="count"><%= tag_alias.antecedent_tag.post_count rescue 0 %></span></td>
-        <td class="category-<%= tag_alias.consequent_tag.try(:category) %>"><%= link_to tag_alias.consequent_name, posts_path(:tags => tag_alias.consequent_name) %> <span class="count"><%= tag_alias.consequent_tag.post_count rescue 0 %></span></td>
-        <td>
-          <% if tag_alias.forum_topic_id %>
-            <%= link_to tag_alias.forum_topic_id, forum_topic_path(tag_alias.forum_topic_id) %>
-          <% end %>
-        </td>
-        <td><%= link_to_user(tag_alias.approver) if tag_alias.approver %></td>
-        <td id="tag-alias-status-for-<%= tag_alias.id %>">
-          <%= tag_alias.status %>
-        </td>
-        <td>
-          <%= link_to "Show", tag_alias_path(tag_alias) %>
+<%= table_for tag_aliases, {class: "striped", width: "100%"} do |t| %>
+  <% t.column "From", {width: "25%"} do |tag_alias| %>
+    <span class="category-<%= tag_alias.antecedent_tag.try(:category) %>"><%= link_to tag_alias.antecedent_name, posts_path(:tags => tag_alias.antecedent_name) %> <span class="count"><%= tag_alias.antecedent_tag.post_count rescue 0 %></span></span>
+  <% end %>
+  <% t.column "To", {width: "25%"} do |tag_alias| %>
+    <span class="category-<%= tag_alias.consequent_tag.try(:category) %>"><%= link_to tag_alias.consequent_name, posts_path(:tags => tag_alias.consequent_name) %> <span class="count"><%= tag_alias.consequent_tag.post_count rescue 0 %></span></span>
+  <% end %>
+  <% t.column "Reference", {width: "10%"} do |tag_alias| %>
+    <% if tag_alias.forum_topic_id %>
+      <%= link_to tag_alias.forum_topic_id, forum_topic_path(tag_alias.forum_topic_id) %>
+    <% end %>
+  <% end %>
+  <% t.column "Approver", {width: "15%"} do |tag_alias| %>
+    <%= link_to_user(tag_alias.approver) if tag_alias.approver %>
+  <% end %>
+  <% t.column "Status", {width: "5%"} do |tag_alias| %>
+    <span id="tag-alias-status-for-<%= tag_alias.id %>"><%= tag_alias.status %></span>
+  <% end %>
+  <% t.column "", {width: "15%"} do |tag_alias| %>
+    <%= link_to "Show", tag_alias_path(tag_alias) %>
 
-          <% if tag_alias.editable_by?(CurrentUser.user) %>
-            | <%= link_to "Edit", edit_tag_alias_path(tag_alias) %>
-          <% end %>
+    <% if tag_alias.is_pending? && tag_alias.editable_by?(CurrentUser.user) %>
```

Comment:
Is adding the `tag_alias.is_pending?` check necessary here? Wasn't present before and it should already be handled by `editable_by?` (see `editable_by?` in `app/models/tag_relationship.rb`).

Reply:
Oh, I see it was for symmetry with tag implications. In that case it's fine (both checks could be removed in a later commit though).